### DISCUSSION TITLE
All Keycloak >=18.0.0 images have security findings >= high?

### DISCUSSION THREAD

**Discussion by jfrantzius** [keycloak/keycloak#13666] [Q&A]
Hi,
our Azure Docker image repository reported security findings >= High for our Keycloak 18.0.1 Docker image, so I started searching for security fixes. Now when looking at https://quay.io/repository/keycloak/keycloak?tab=tags and clicking "Expanded", it turns out that all Keycloak Docker images >=18.0.0 have security findings with severity >= High. 

That makes me wonder how we could possibly get hold of a Keycloak image with security findings <= Medium?

I haven't found out yet how exactly those images on quay.io are built, and what their base image is in particular, as quay.io doesn't refer to source code of a Dockerfile. Does Keycloak use a Red Hat UBI image as its base image? That could mean that this could be fixed by using a newer base UBI image?

**Comment by jfrantzius** [comment]
I allowed myself to file https://github.com/keycloak/keycloak/issues/13701 for this

**Comment by abstractj** [comment]
@jfrantzius answered on https://github.com/keycloak/keycloak/issues/13701
